conda_forge_tick/migrators/version.py

```diff
@@ -197,21 +222,27 @@ def migrate(
         **kwargs: Any,
     ) -> "MigrationUidTypedDict":
         version = attrs["new_version"]
-
-        with open(os.path.join(recipe_dir, "meta.yaml")) as fp:
-            raw_meta_yaml = fp.read()
-
-        updated_meta_yaml, errors = update_version(
-            raw_meta_yaml,
-            version,
-            hash_type=hash_type,
-        )
+        recipe_dir = Path(recipe_dir)
+        recipe_file = recipe_dir / "meta.yaml"
+        recipe_yaml = recipe_dir / "recipe.yaml"
+        if recipe_file.exists():
+            raw_meta_yaml = recipe_file.read_text()
+            updated_meta_yaml, errors = update_version(
+                raw_meta_yaml,
+                version,
+                hash_type=hash_type,
+            )
+        elif recipe_yaml.exists():
+            recipe_file = recipe_yaml
+            updated_meta_yaml, errors = update_version_v1(
```

Comment:
This routine calls into jinja2 and so needs to be executed in a container. This is not happening here AFAICT.

Reply:
This code is essentially equivalent to `update_version` which is called below, outside a container etc. (uses the same Jinja eval methods).

Reply:
Ah I see. The migrator is run in a container so that covers this case. However, because of the way this PR is done currently, the admin commands to update recipe versions won't work for v1. That will need to be fixed if someone wants to use that.